### DISCUSSION TITLE
Do I need to install WeMod separately for every game I want to use it on?

### DISCUSSION THREAD

**Discussion by PKMNNick9403** [DeckCheatz/wemod-launcher#82] [Q&A] [answered]
Prefacing this by saying I am VERY inexperienced with Linux, but I just wanted to make sure I understood the guide correctly. Do I need to do the entire guide again for each game I want to use WeMod on, or does it work for multiple games?

**Comment by trippixn963** [comment]
Yes

**Comment by marvin1099** [comment, marked as answer]
As john has pointed out yes,
but also wen using wemod on a other game it will copy the working wemod files to the new game so, its faster then the first time installing.
